packages/editor/src/components/block-settings-menu/reusable-block-convert-button.js

```diff
@@ -29,6 +30,7 @@ export function ReusableBlockConvertButton( {
 				<MenuItem
 					className="editor-block-settings-menu__control"
 					icon="controls-repeat"
+					disabled={ ! canCreateBlocks }
```

Comment:
I think this is the only block settings menu item that ends up disabled. Most of the others are just not rendered when they're not valid in a specific context and I think it's worth following the same pattern here. The same pattern (hiding instead of disabling) is also used for other bits of UI when users don't have the capability.

Reply:
> The same pattern (hiding instead of disabling) is also used for other bits of UI when users don't have the capability.

Makes sense to be consistent here then 🙂 — have updated the behaviour in 708a50484fc99537a5f3368b4dbb66e134198741.